pkgs/development/libraries/libusb1/default.nix

```diff
@@ -13,12 +13,15 @@ stdenv.mkDerivation (rec {
 
   nativeBuildInputs = [ pkgconfig ];
   propagatedBuildInputs =
-    stdenv.lib.optional stdenv.isLinux systemd ++
+    stdenv.lib.optional (stdenv.isLinux && !stdenv.hostPlatform.isMusl) systemd ++
     stdenv.lib.optionals stdenv.isDarwin [ libobjc IOKit ];
 
   NIX_LDFLAGS = stdenv.lib.optionalString stdenv.isLinux "-lgcc_s";
 
-  preFixup = stdenv.lib.optionalString stdenv.isLinux ''
+  configureFlags =
+    stdenv.lib.optional stdenv.hostPlatform.isMusl "--disable-udev";
+
+  preFixup = stdenv.lib.optionalString (stdenv.isLinux && !stdenv.hostPlatform.isMusl) ''
```

Comment:
Please give the expression `stdenv.isLinux && !stdenv.hostPlatform.isMusl` an explicit name, like you did in https://github.com/NixOS/nixpkgs/pull/71560/files#diff-85094d104476de3887410bbe74b0de87R57, because that prevents that people accidentally update only 1 of the 2 locations in the future.

Reply:
I have fixed this in 0262985 as suggested in https://github.com/NixOS/nixpkgs/pull/71575#issuecomment-544782754.